templates/search/blocks/visualisation_menu.html

```diff
@@ -1,16 +1,18 @@
+{% load search_tags %}
+
 <!--visualisation secondary navigation-->
 <div class="tna-container">
     <div class="tna-column tna-column--full">
         <div class="search-visualisation-nav">
             <ul class="search-visualisation-nav__list" data-id="search-visualisation-nav-list">
                 <li class="search-visualisation-nav__list-item">
-                    <a href="/" class="search-visualisation-nav__link" aria-current="true">
+                    <a href="?{% query_string_include 'views' views.LIST.value %}" class="search-visualisation-nav__link"{% if form.views.value == views.LIST.value %} aria-current="true"{% endif %}>
```

Comment:
1. Is List Map view in scope for this ticket?
2. What is the expected behaviour/url link of List View, when a Search criteria are applied on a page and the user then navigates to Map view?

Reply:
Discussed in stand up, a separate ticket will be created to address the navigation behaviour